Quotient/database.cpp

```diff
@@ -238,6 +239,41 @@ void Database::migrateTo9()
     commit();
 }
 
+void Database::migrateTo10()
+{
+    qCDebug(DATABASE) << "Migrating database to version 10";
+
+    transaction();
+
+    execute(QStringLiteral("ALTER TABLE inbound_megolm_sessions ADD senderClaimedEd25519Key TEXT;"));
+
+    auto query = prepareQuery("SELECT senderKey FROM inbound_megolm_sessions;"_ls);
+    execute(query);
+
+    QSet<QString> keys;
+    while (query.next()) {
+        keys += query.value("senderKey"_ls).toString();
+    }
+    for (const auto& key : keys) {
+        auto query = prepareQuery("SELECT edKey FROM tracked_devices WHERE curveKey=:curveKey;"_ls);
+        query.bindValue(":curveKey"_ls, key);
+        execute(query);
+        if (!query.next()) {
+            continue;
+        }
+        const auto &edKey = query.value("edKey"_ls).toByteArray();
```

Comment:
Avoiding variables shadowing each other. Eventually, I hope we'll get a generic method to retrieve a single value from the database - we've already got a lot of boilerplate code doing that. Not necessarily for this PR though.
```suggestion
        auto edKeyQuery = prepareQuery("SELECT edKey FROM tracked_devices WHERE curveKey=:curveKey;"_ls);
        edKeyquery.bindValue(":curveKey"_ls, key);
        execute(edKeyQuery);
        if (!edKeyQuery.next()) {
            continue;
        }
        const auto &edKey = edKeyQuery.value("edKey"_ls).toByteArray();
```